scripts/app_diff.sh

```diff
@@ -16,5 +16,12 @@ SRC_APP_DIR=$EMBOX_DIR/$3 #third-party/fuse/fuse
 cd $EXTBLD_APP_DIR
 mkdir -p $APP_NAME-orig
 tar xzf $EMBOX_DIR/download/${APP_NAME}.tar.gz -C $APP_NAME-orig --strip-components=1
-diff -aur -x configure $APP_NAME-orig $APP_NAME | grep -v "^Only in" | tee $SRC_APP_DIR/patch.txt
-rm -rf $APP_NAME-orig
+diff -aur -x configure ../$APP_NAME-orig $APP_NAME | grep -v "^Only in" | tee $SRC_APP_DIR/patch.txt
```

Comment:
What I mean is `diff -Naur -x configure ../$APP_NAME-orig $APP_NAME | tee $SRC_APP_DIR/patch.txt`. But as far as I can see you've just reverted changes by commit e6a534b834a563588098cf2ec7e9982393910011.